03/app.js

```diff
@@ -28,6 +29,23 @@ if (newButton) {
       newButton.style[setting] = buttonSettings.css[setting];
     }
   }
+}*/
+
+if (newButton) {
+  for (const set in buttonSettings) {
+    if (typeof set === "string") {
```

Comment:
Raczej `typeof buttonSettings[set] === 'string'` i wew. tego if-a ma być tylko `newButton.innerHTML = buttonSettings[set];`, a potem wykorzystać dostępne rozwiązanie:
```
for (const setting in buttonSettings[set]) {
       if(set === 'attr') {
           newButton.setAttribute(setting, buttonSettings.[set][setting]);
      }
}
```

PS. To jest tylko kod instruktarzowy - pewnie nie zadziała prawidłowo po wklejeniu ;)